src/server/web/cassh_web.py

```diff
@@ -198,7 +197,7 @@ def send(current_user=None):
     payload = {}
     payload.update({'realname': current_user['name'], 'password': current_user['password']})
     payload.update({'username': username})
-    payload.update({'pubkey': quote_plus(pubkey.read().decode('UTF-8'))})
+    payload.update({'pubkey': pubkey.read().decode('UTF-8')})
```

Comment:
Why removing the `quote_plus` ?

Reply:
I guess the framework is already quoting the text.

Reply:
ok